dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/config/UnitTestConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package org.hisp.dhis.config;
+
```

Comment:
Needs source code license and @author info.

Reply:
done